docs/llb/llb-api-spec.md

```diff
@@ -705,6 +707,8 @@ The announcements were retrieved successfully.
       "id": ID,
       "title": STRING,
       "description": STRING,
+      "eventId": ID OR NULL,
```

Comment:
I believe that this route does not include the `eventId`, as it gets site-wide announcements that are not associated with an event. If you observed something different, what `eventId` is returned?

Reply:
Yeah it does:
![image](https://user-images.githubusercontent.com/46979800/109894088-d5c09100-7c5a-11eb-9f3c-4b40d429c0f9.png)